Update instructions.md (#1613)
exercism/javascript: exercises/practice/resistor-color-duo/.docs/instructions.append.md

# Instructions append

## Notes

> In another word based on the example provided above, the input should be an `array` of color names and the output should be a two-digit `number` 
> 
> `[‘brown’, ‘green’]` should return `15`
> 
> `[‘brown’, ‘green’, ‘violet’]` should return `15`, ignoring the third color.

Reviewer: These notes are rendered directly below the instructions text so there is no need to repeat the example. The important point is not that the third color should be ignored. The important point is that we clarify the input and output in general.

```suggestion
The provided input will be an array of color names and the output should be a number.
```
